packages/@lightningjs/ui-components/src/components/Base/Base.d.ts

```diff
@@ -63,6 +63,8 @@ declare class Base<
    */
   loaded?: Promise<void>;
 
+  // TODO: flag for discussion
+  // internal stuff can be omitted --> `shouldSmooth` is internal?
```

Comment:
That's a good callout 🤔 I think since it's not underscored, it might be worth adding here. I think we had to use it once or twice in Flex for some Row/Column updates.

Reply:
so my next question is, should `shouldSmooth` get added to `Base` since technically it can be set on any component?

Reply:
Did we ever discuss this? I think it makes sense to add to Base.

Reply:
i double checked and this exists in Base, however only in the class body and without any actual documentation. Opened a PR to update those docs https://github.com/rdkcentral/Lightning-UI-Components/pull/266